client/client.go

```diff
@@ -395,6 +395,15 @@ type RemoteContext struct {
 	// MaxConcurrentDownloads is the max concurrent content downloads for each pull.
 	MaxConcurrentDownloads int
 
+	// MaxConcurrentDownloadOperations is the max concurrent download operations for each pull.
+	MaxConcurrentDownloadOperations int
+
+	// MaxConcurrentDownloadsPerLayer is the max concurrent downloads per layer for each pull.
+	MaxConcurrentDownloadsPerLayer int
```

Comment:
Can we add some explanation about the difference among the 3 options, especially the first two options? thanks.

E.g., my understading is that before this PR, an image pull will simply have `MaxConcurrentDownloads` number of concurrent layer download (one for each layer). What will be the behavior after thie PR?

Reply:
Why do we need MaxConcurrentDownloadOperations?

Reply:
I updated the docs ! LMK if that helps better !